### PR TITLE
Adds more descriptive and helpful error messages to the NFT contracts and txs

### PR DESCRIPTION
**Pull request by joshuahannan**
Closes: #232

## Description

* Updates all error messages to include more information and relevant data about the error if possible
* Updates CLI commands for C1.0

______

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

File: contracts/ExampleNFT.cdc [hunk 1]
```diff
@@ -133,7 +133,7 @@ access(all) contract ExampleNFT: NonFungibleToken {
                             resourceType: nil,
                             viewType: Type<MetadataViews.EVMBridgedMetadata>()
                         ) as! MetadataViews.EVMBridgedMetadata?
-                        ?? panic("Could not resolve contract-level EVMBridgedMetadata")
+                        ?? panic("Could not resolve contract-level EVMBridgedMetadata.")
```
Comment (bjartek) [inline contracts/ExampleNFT.cdc:136]: Imho resolving a view should never
Panic. Return nil.

**Review by sisyphusSmiling — APPROVED** [review]
Really like the detailed error message, thanks for fixing these up! My main suggestion is not a blocker, but it would be great to see consistent formatting e.g. some messages wrap values in `()` or `<>` and others prefix values with `=`. In reality, this probably more of a preference/nitpick

File: contracts/ExampleNFT.cdc (hunk 1 above)
Comment (joshuahannan) [reply contracts/ExampleNFT.cdc:136]: Agreed! I'll remove that. Good catch!

**Comment by joshuahannan** [comment]
@sisyphusSmiling Good point! I'll look through and make sure they are consistent. The only time I use `<>` is when I am dealing with types because that is usually what they are contained in in the code, so I thought it would make sense to be consistent with that. But agreed about using `=` vs `()`. I'll find a consistent format.

**Pull request merged**